perf/ftdc_rollups.go

```diff
@@ -71,6 +71,11 @@ func CreatePerformanceStats(dx *ftdc.ChunkIterator) (*PerformanceStatistics, err
 					perfStats.timers.extractedDurations,
 					extractValues(convertToFloats(metric.Values), lastValue)...,
 				)
+				// In order to avoid memory panics, reject
+				// anything larger than 4GB.
+				if len(perfStats.timers.extractedDurations) > 500000000 {
```

Comment:
is it still possible to run into issues if multiple jobs with large files were to land on the same app server? Would you consider using [MemStats](https://golang.org/pkg/runtime/#MemStats) to find how much memory the application is using in total?